readdy/main/model/observables/Reactions.cpp

```diff
@@ -89,6 +89,7 @@ void Reactions::append() {
 }
 
 void Reactions::initialize(Kernel *const kernel) {
+    // fixme why is this warning here? e.g. virial silently sets its corresponding flag in context
```

Comment:
for one it is an simulation parameter that impacts performance and a side effect that is not very obvious from just registering an observable - I blame our software design for that :-)

For consistency sake we should either remove it or add it to the virial stuff too I guess.

Reply:
I see. Then I think the user shall be informed in the virial as well.